packages/clerk-js/src/utils/injectedWeb3Providers.ts

```diff
@@ -27,12 +27,11 @@ interface EIP6963ProviderDetail {
 }
 
 type EIP6963AnnounceProviderEvent = CustomEvent;
-type InjectedWeb3Provider = MetamaskWeb3Provider | CoinbaseWeb3Provider;
+type InjectedWeb3Provider = MetamaskWeb3Provider;
```

Comment:
❓ We don't need `CoinbaseWalletWeb3Provider` here?

Reply:
No because CoinbaseWallet is not an injected provider but it's handled via the coinbase wallet sdk